src/backend/cgobj.c

```diff
@@ -373,6 +373,9 @@ void too_many_symbols()
 {
 #if SCPP
     err_fatal(EM_too_many_symbols, 0x7FFF);
+#elif TARGET_WINDOS // COFF
+    error(NULL, 0, 0, "more than %d sections in object file", 65279);
```

Comment:
What's that number? Google wouldn't find anything with this number and OMF or COFF.